src/ros_bridge/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM osrf/ros:foxy-desktop
+
+WORKDIR v-hsc-ros-bridge/
+
+# install ros build tools
+RUN apt-get update && apt-get install -y --no-install-recommends \
+      python3-colcon-common-extensions libopencv-dev liblua5.2-dev protobuf-compiler libprotobuf-dev && \
+    rm -rf /var/lib/apt/lists/*
+
+# clone ros package repo
+RUN git clone \
+      -b main \
+      https://github.com/robocup-hl-tc/v-hsc-ros-bridge.git
```

Comment:
Now that I think about it. I think it would be better if you used a `COPY` command instead of `RUN git clone` because this Dockerfile is supposed to serve as a template for other teams and teams probably don't want to use a `git clone` but a `COPY`, so that also local changes are considered (not only changes pushed to the master)